### DISCUSSION TITLE
Blobfuse2 default encryption used for data transfer

### DISCUSSION THREAD

**Discussion by pklein-rescale** [Azure/azure-storage-fuse#1092] [Q&A] [answered]
Hi,

I'm hoping to confirm something about how Blobfuse2 handles encryption -- I assume it utilizes HTTPS (TLS 1.2) for secure data transmission by default, given its reliance on the Azure Storage C++ Lite SDK and general security practices on Azure.  

Can someone on the development team confirm that this is true?  Does this depend on the "Secure transfer required" setting on the Azure storage account itself?  This may be something worth explicitly addressing in the documentation.

Thanks very much,

Pete

**Comment by bhansalivikas** [comment, marked as answer]
Yes your understanding is correct. Blobfuse uses HTTPS as underlying protocol to transfer data so encryption is inherent. There is a way to configure it to use HTTP though. No need to configure anything on account side, most accounts by default will have HTTPS enable.

**Reply by pklein-rescale** [reply]
Thanks very much.
